src/app/components/data-entry-grid/index.html

```diff
@@ -233,6 +238,13 @@ <h2>Passing cell editor parameters</h2>
         You can pass inputs to the SKY UX components used in cell editors via the <a href="https://www.ag-grid.com/javascript-grid-column-properties/">column definition's <sky-code>cellEditorParams</sky-code> property.</a> The component properties are nested under <sky-code>skyComponentProperties</sky-code>. All component properties should be supported, and you can see the defined types by importing <sky-code>SkyCellEditorDatepickerProperties</sky-code> or <sky-code>SkyCellEditorAutocompleteProperties</sky-code>. <sky-code>cellEditorParams</sky-code> can also be a function that returns a param object for dynamic editing params. See the demo for examples of using an object or a function.
       </p>
 
+      <h2>Using other SKY UX components in columns</h2>
+      <p>
+        There are some occasions where you need full control over the SKY UX component being rendered in the cell. In this case, you can create your own <a href="https://www.ag-grid.com/javascript-grid-cell-rendering-components/">cell renderer</a> and place a SKY UX component inside of it. Here are some cases where you will want to do this:
+      </p>
+      <ul>
+        <li>Context menu - Creating your own context menu component gives you full control over the text, menu options, and click actions. See the demo for an example.</li>
```

Comment:
I'd remove "Here are some cases..." and start a new line instead with the suggestion below. We can always go back to a bulleted list if more items are added, but we can ditch the list for now because there's only one example listed.

For example, use a cell renderer when creating your own context menu component for full control of the text, menu options, and click actions.

Reply:
So the big goal for this section is to call out that you _need_ to make your own context menu. I think a lot of people expect it to be provided for them, but it is too complex of a component to make one that works for everyone. They don't see it in the docs, so they assume that ag-grid just "doesn't have context menu". The reason I made the list is because I think it draws the attention to that a bit more, instead of just being stuck in a paragraph that gets skimmed over. I also think there will be more components added to that list soon. 

Do you think there is a way to make it more clear that you can use a context menu, you just have to make it yourself? Or just putting it in that blurb will suffice?

Reply:
It still feels a little off to use a bulleted list for one item, but I get wanting to quickly draw attention to it. Because that list will grow soon, that makes me more inclined to keep the list. I'll let @blackbaud-johnly weigh in too.

If we switch back to original format, is saying something like, "You must create your own context menu to fully control the text, menu options..." too strong of a prescription? In the original, I didn't quite take away that you _needed_ to but should.

Reply:
I'd delete the "Here are some cases...` sentence and also delete the bullet for now as Denise suggested. As she pointed out, we can always convert back to a bullet list when we have more items. I get the point that we intend to add more bullets, but I'd wait until we have those additional items before making this a bullet list.

As for the point about users _needing_ to make their own context menu, I agree with Denise that I don't get that point from the existing docs. I suggested an update to the previous paragraph to make this clear generically, so hopefully that works.

As for the content in the existing bullet, what do you think about rephrasing this a bit to emphasize the point about needing to make their own context menu component and state that it is an example so that we imply that the same is true for other components? I'm thinking of something like, `For example, to include a context menu with full control over text, menu options, and click actions, you create a context menu component and cell renderer and place the component in the cell renderer. See the demo for an example.` What do you think?